src/Components/test/testassets/BasicTestApp/FormsTest/ExperimentalValidationComponent.razor

```diff
@@ -0,0 +1,181 @@
+@using System.ComponentModel.DataAnnotations
+@using Microsoft.AspNetCore.Components.Forms
+
+
```

Comment:
Can you add a paragraph here describing the how this component is used in the test? We started doing this to more easily understand other people's E2E tests.

Just a `<p>...</p>` here would do. Specially given how big and complex this one is.